src/theme/components/socialmedialinks.ts

```diff
@@ -149,10 +154,9 @@ const SocialMediaLinks = defineStyleConfig({
       };
     },
     circular(props: StyleFunctionProps) {
-      const { color, size, layout } = props;
+      const { color, size } = props;
       let theseColors = getColor(color);
       return {
-        flexDirection: { base: "column", md: layout },
         gap: "xs",
```

Comment:
Same as above, update the `gap` to use the following:
`gap: { base: "0", md: "xs" },`